azure-pipelines.yml

```diff
@@ -439,6 +439,10 @@ stages:
               vs_release:
                 _configuration: Release
                 _testKind: testVs
+              # experimental_features:
```

Comment:
Honestly I am not a fan adding more commented code to our pipelines.

Reply:
Yeah I can revert this. It's more for reference, but not the best solution anyway.